Sprint-1/exercises/decimal.js

```diff
@@ -7,3 +7,13 @@ const num = 56.5678;
 // Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
 
 // Log your variables to the console to check your answers
+
+let wholeNumberPart = Math.round(num);
+    console.log(wholeNumberPart)
+
+let decimalPart = num - Math.floor(num);
+    console.log(decimalPart);
```

Comment:
If you run the program, you may notice that `console.log()` outputs the value of `decimalPart` 
with more than 4 decimal places (which is normal because of floating point arithmetic). 

If you were asked to print the value of `decimalPart` to exactly 4 decimal places,
how would you modify the code?

Reply:
let decimalPart = (num - Math.floor(num)).toFixed(4);
    console.log(decimalPart);